src/tinyformat.h

```diff
@@ -1152,6 +1154,12 @@ TINYFORMAT_FOREACH_ARGNUM(TINYFORMAT_MAKE_FORMAT_FUNCS)
 template<typename... Args>
 std::string format(const std::string &fmt, const Args&... args)
 {
+    // Numerous tfm::format(...) callers are implicitly assuming that the output
+    // returned is locale independent. Notable examples include i64tostr, itostr
+    // and ScriptToAsmStr. Make sure to do a thorough review of all call sites
+    // (and other direct/indirect stringstream consumers) if this assumption is
+    // ever removed in the future.
+    assert(std::locale{} == std::locale::classic());
```

Comment:
I'd rather just make this explicitly use the classic locale than forcing a global assumption on the code (i.e. `oss.imbue(std::locale::classic());`).

Reply:
Fixed!

Reply:
Agree with @sipa here.